addons/l10n_ec_ats/security/ir.model.access.csv

```diff
@@ -0,0 +1,9 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink

```

Comment:
It's important that you read the guidelines before you continue your development.
https://www.odoo.com/documentation/9.0/reference/guidelines.html

Module structure
Directories

A module is organised in important directories. Those contain the business logic; having a look at them should make understand the purpose of the module.

    data/ : demo and data xml
    models/ : models definition
    controllers/ : contains controllers (HTTP routes).
    views/ : contains the views and templates
    static/ : contains the web assets, separated into css/, js/, img/, lib/, ...

Other optional directories compose the module.

    wizard/ : regroups the transient models (formerly osv_memory) and their views.
    report/ : contains the reports (RML report [deprecated], models based on SQL views (for reporting) and other complex reports). Python objects and XML views are included in this directory.
    tests/ : contains the Python/YML tests